src/parsers/manifestjson.js

```diff
@@ -185,14 +125,12 @@ export default class ManifestJSONParser {
         this.parsedJSON.applications.gecko &&
         this.parsedJSON.applications.gecko.update_url) {
       this.collector.addError(messages.MANIFEST_UPDATE_URL);
-      isValid = false;
+      this.isValid = false;
     }
 
     if (isToolkitVersionString(this.parsedJSON.version)) {
       this.collector.addNotice(messages.PROP_VERSION_TOOLKIT_ONLY);
     }
-
-    return isValid;
```

Comment:
Nope. Or not any more; it was a bit convoluted, where `ManifestJSONScanner.isValid` would be set but then also set using this... I just tidied it up.